CHANGELOG.rst

```diff
@@ -20,6 +21,18 @@ Major Features
     so this change should not break any existing consumers
     unless you are specifically relying on the output being an instance of :class:`tuple`.
 
+Deprecations
+------------
+
+* Deprecate master key providers in favor of keyrings.
+
+    * We do not have a planned version for when we will remove master key providers.
```

Comment:
This is a little pessimistic, how about something like "Master key providers continue to be supported. Prior to their removal, we will communicate this as defined in our versioning policy."